scripts/sct_label_vertebrae.py

```diff
@@ -377,7 +401,21 @@ def main(args=None):
              'NearestNeighbor'),
             verbose=verbose,
             is_sct_binary=True,
-           )
+            )
+
+    # un-straighten posterior disc map
+    # it won't exist if we don't use the detection since it is based on the network prediction
```

Comment:
Nitpick: This comment could be a little clearer:
```suggestion
    # disc_posterior_tmp.nii.gz only exists as a result of automatic c2c3 detection, which
    # only happens if fname_disc isn't provided.
```